eng/Subsets.props

```diff
@@ -219,6 +219,7 @@
     <SubsetName Include="Clr.PalTestList" OnDemand="true" Description="Generate the list of the CoreCLR PAL tests. When using the command line, use Clr.PalTests instead." />
     <SubsetName Include="Clr.Hosts" Description="The CoreCLR corerun test host." />
     <SubsetName Include="Clr.Jit" Description="The JIT for the CoreCLR .NET runtime." />
+    <SubsetName Include="Clr.WasmJit" Description="The cross-targeting WebAssembly JIT compiler for the CoreCLR .NET runtime." />
```

Comment:
We may want to think about how to clearly message things if we ever add a client-side (HOST_WASM) wasm jit in the future, since the current naming makes it sound like that's what this is. (I think the description is perfectly clear, though.)

It'll probably be sufficient to just name it something like 'Clr.WasmClientJit' or something if it happens, but maybe it's worth explicitly naming the current one something like 'Clr.WasmCompiler' or 'Clr.WasmAot'?